README.md

```diff
@@ -224,6 +224,24 @@ Sends battery information to the server.
 | 6        | Desconnection                | 0                   |
 | 2        | Sleeping value (unused)      | 0$                  |
 
+#### BT
+The BT module is meant to be used in the Generic Button interactions of the Thinking Things IOT Agent. It works slightly different as the typical TT Module, as it will be described along this section. The available attributes are the following:
+
+| Position | Meaning                      | Possible values     |
+|:--------:|:---------------------------- |:------------------- |
+| 1        | Operation                    | S, C, P, X          |
+| 2        | Action                       | 1                   |
+| 3        | Extra or request ID          | 16234               |
+
+The Generic button can work on two different modes, making use of different operations in each one. In **synchronous mode**, all the information sent by the button is immediatly sent to the Third Party systems through the Context Broker, and the IOTA waits for its return, to informa the device about the success or error of the request. This synchronous interaction is performed using the **S** operation, and it beguins and end with its execution. 
```

Comment:
Not that I like the Black Button name, as it doesn't really mean anything (White or Blank button on the other hand would make much more sense).